File: GTTS/INL.TrainingService.Models/GetTrainingEventParticipantAttachment_Item.cs
using System;

namespace INL.TrainingService.Models
{
    public class GetTrainingEventParticipantAttachment_Item : IGetTrainingEventParticipantAttachment_Item
    {
        public string ParticipantType { get; set; }
        public long TrainingEventParticipantAttachmentID { get; set; }
        public long TrainingEventID { get; set; }
        public long PersonID { get; set; }
        public long FileID { get; set; }
        public int FileVersion { get; set; }
        public int TrainingEventParticipantAttachmentTypeID { get; set; }
        public string TrainingEventParticipantAttachmentType { get; set; }
        public string FileName { get; set; }
        public string FileLocation { get; set; }
        public byte[] FileHash { get; set; }
        public int FileSize { get; set; }
        public byte[] FileContent { get; set; }
        public string ThumbnailPath { get; set; }
        public string Description { get; set; }
        public bool? IsDeleted { get; set; }
        public int ModifiedByAppUserID { get; set; }
        public DateTime ModifiedDate { get; set; }
        public string FileJSON { get; set; }
        public string ModifiedByUserJSON { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/get-training-event-participant-attachment_item.ts


import { IGetTrainingEventParticipantAttachment_Item } from './iget-training-event-participant-attachment_item';

export class GetTrainingEventParticipantAttachment_Item implements IGetTrainingEventParticipantAttachment_Item {
  
	public ParticipantType: string = "";
	public TrainingEventParticipantAttachmentID: number = 0;
	public TrainingEventID: number = 0;
	public PersonID: number = 0;
	public FileID: number = 0;
	public FileVersion: number = 0;
	public TrainingEventParticipantAttachmentTypeID: number = 0;
	public TrainingEventParticipantAttachmentType: string = "";
	public FileName: string = "";
	public FileLocation: string = "";
	public FileHash: number[] = null;
	public FileSize: number = 0;
	public FileContent: number[] = null;
	public ThumbnailPath: string = "";
	public Description: string = "";
	public IsDeleted?: boolean;
	public ModifiedByAppUserID: number = 0;
	public ModifiedDate: Date = new Date(0);
	public FileJSON: string = "";
	public ModifiedByUserJSON: string = "";
  
}






